setup.py

```diff
@@ -22,7 +22,11 @@
 
 name = "google-cloud-bigquery"
 description = "Google BigQuery API client library"
-version = "2.0.0"
+
+version = {}
+with open("google/cloud/bigquery/version.py") as fp:
```

Comment:
I would edit this as I did in https://github.com/googleapis/python-datastore/pull/97. The assumption of separator and relative pathing seemed a bit fragile, so I hardened it a bit.